src/block_cache/block_chunkcache.c

```diff
@@ -371,7 +371,6 @@ __chunkcache_tmp_hash(WT_CHUNKCACHE *chunkcache, WT_CHUNKCACHE_HASHID *hash_id,
     uint64_t hash_final;
 
     WT_CLEAR(intermediate);
-    intermediate.name_hash = __wt_hash_city64(object_name, strlen(object_name));
```

Comment:
Ah, accidental removal?

Reply:
It did not seem to be used at all, see this [comment](https://github.com/wiredtiger/wiredtiger/pull/9719#discussion_r1348243436).

Reply:
Oh yes my bad, I removed the wrong one!

Reply:
Reverted ba6bd99f2298d9720c00f1316844ab50eaac0fde